packages/ast/api.d.ts

```diff
@@ -65,6 +65,10 @@ declare interface XMLElement {
     // Will only exist if there is a valid "Closing Name" for the XML Element
     // - Note that this is an **optional** syntax in a valid XML.
     readonly closeName?: XMLToken;
+    // Will only exist if there is a valid "Start Name" for the XML Element

```

Comment:
This does not describe what this property actually represents.
Maybe a small XML code snippet explaining that this is the **RANGE** of the element opening + attributes section should be added.